docs/using-hive.md

```diff
@@ -558,6 +560,73 @@ openstack:
   flavor: m1.large
 ```
 
+#### Configuring Availability Zones
+
+The desired Availability Zones (AZ) to create new worker nodes in can be specified in the `MachinePool` YAML (`spec.platform.<provider>.zones`), for example:
+
+```yaml
+apiVersion: hive.openshift.io/v1
+kind: MachinePool
+metadata:
+  name: mycluster-worker
+  namespace: mynamespace
+spec:
+  clusterDeploymentRef:
+    name: mycluster
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        iops: 100
+        size: 22
+        type: gp2
+      type: m4.xlarge
+      zones:
+        - us-east-1a
+        - us-east-1b
+  replicas: 3
+```
+
+If the Availability Zones are not configured in the `MachinePool`, then all of the AZs in the region will be used and a `MachineSet` resource will be created in all of the AZs.
+
+#### Auto-scaling
+
+`MachinePools` can be configured to auto-scale to scale up or down worker nodes as needed based on resource utilization of the deployed cluster by creating a `ClusterAutoscaler` on the deployed cluster.
+
+```yaml
+apiVersion: hive.openshift.io/v1
+kind: MachinePool
+metadata:
+  name: mycluster-worker
+  namespace: mynamespace
+spec:
+  clusterDeploymentRef:
+    name: mycluster
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        iops: 100
+        size: 22
+        type: gp2
+      type: m4.xlarge
+  autoscaling:
+    minReplicas: 3
+    maxReplicas: 6
+```
+
+The number of minimum replicas must be equivalent to the number of configured Availability Zones.
+
+The `spec.replicas` and `spec.autoscaling` configurations cannot be configured simultaneously.
+
+The `spec.autoscaling.maxReplicas` is an optional field, if it is not configured, then nodes will be auto-scaled without restriction based on resource utilization needs.
+
+##### Integration with Horiztonal Pod Autoscalers
+
+A `MachinePool` configured to auto-scaling mode creates a `ClusterAutoscaler` on the deployed cluster. `ClusterAutoscalers` can co-exist and work with Horiztonal Pod Autoscalers to ensure that there are enough available nodes to meet the auto-scaled pod replica count requirements. See excerpt from OpenShift documentation:
```

Comment:
Can this be linked to said documentation?

Reply:
Hey Eric, thanks for the quick reviews.  I can change the copy/paste to a link, but I wasn't sure if there would be some concern about the link getting stale since it would be pinned to a specific OCP version doc.

Reply:
Nono, keep the quote -- just add the link as well.

(Agree about stuff getting stale, it's always a problem.)